### DISCUSSION TITLE
Change Default 'p' in enable_cell_picking

### DISCUSSION THREAD

**Discussion by emielke12** [pyvista/pyvista#2257] [Q&A]
I'm implementing a picking routine and find that it's pretty annoying to be using the mouse in my right hand and have to move my left hand from its resting spot on the left side of the keyboard all the way over to hit the 'p' button to select a point on the mesh. Is there any way to change this default key press?

**Comment by banesullivan** [comment]
Try the new `left_clicking ` option from https://github.com/pyvista/pyvista/pull/2266
